source/simulator/assemblers/stokes.cc

```diff
@@ -495,6 +495,76 @@ namespace aspect
     }
 
 
+    template <int dim>
+    void
+    StokesHydrostaticCompressionTerm<dim>::
+    execute (internal::Assembly::Scratch::ScratchBase<dim>   &scratch_base,
+             internal::Assembly::CopyData::CopyDataBase<dim> &data_base) const
+    {
+      internal::Assembly::Scratch::StokesSystem<dim> &scratch = dynamic_cast<internal::Assembly::Scratch::StokesSystem<dim>& > (scratch_base);
+      internal::Assembly::CopyData::StokesSystem<dim> &data = dynamic_cast<internal::Assembly::CopyData::StokesSystem<dim>& > (data_base);
+
+      // assemble RHS of:
+      //  - div \mathbf{u} = \frac{1}{\rho} * \frac{\partial rho}{\partial p} \rho \mathbf{g} \cdot \mathbf{u}
+      // + \frac{1}{\rho} * \frac{\partial rho}{\partial T} \nabla T \cdot \mathbf{u}
```

Comment:
Same here.

Reply:
And the alpha term has the wrong sign (it should be `- \frac{1}{\rho} \frac{\partial rho}{\partial T} \nabla T \cdot \mathbf{u}`). See the implementation (which has the correct sign).